apps/condo/domains/meter/schema/RegisterMetersReadingsService.js

```diff
@@ -0,0 +1,257 @@
+/**
+ * Generated by `createservice meter.RegisterMetersReadingsService --type mutations`
+ */
+const dayjs = require('dayjs')
+const { get, isUndefined, isEmpty, isNumber, isString } = require('lodash')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { GQLCustomSchema, find, getByCondition, getById } = require('@open-condo/keystone/schema')
+
+const { PropertyResolver } = require('@condo/domains/billing/schema/resolvers')
+const access = require('@condo/domains/meter/access/RegisterMetersReadingsService')
+const { IMPORT_CONDO_METER_READING_SOURCE_ID } = require('@condo/domains/meter/constants/constants')
+const {
+    TOO_MUCH_READINGS,
+    ORGANIZATION_NOT_FOUND,
+    PROPERTY_NOT_FOUND,
+    INVALID_METER_VALUE,
+} = require('@condo/domains/meter/constants/errors')
+const { Meter, MeterReading } = require('@condo/domains/meter/utils/serverSchema')
+
+const READINGS_LIMIT = 500
+
+const ERRORS = {
+    TOO_MUCH_READINGS: (sentCount) => ({
+        code: BAD_USER_INPUT,
+        type: TOO_MUCH_READINGS,
+        message: `Too much readings. Maximum is ${READINGS_LIMIT}.`,
+        messageForUser: 'api.meter.registerMetersReadings.TOO_MUCH_READINGS',
+        messageInterpolation: { limit: READINGS_LIMIT, sentCount },
+    }),
+    ORGANIZATION_NOT_FOUND: {
+        code: BAD_USER_INPUT,
+        type: ORGANIZATION_NOT_FOUND,
+        message: 'Organization not found',
+        messageForUser: 'api.meter.registerMetersReadings.ORGANIZATION_NOT_FOUND',
+    },
+    PROPERTY_NOT_FOUND: {
+        code: BAD_USER_INPUT,
+        type: PROPERTY_NOT_FOUND,
+        message: 'Property not found',
+        messageForUser: 'api.meter.registerMetersReadings.PROPERTY_NOT_FOUND',
+    },
+    INVALID_METER_VALUE: (key, value) => ({
+        code: BAD_USER_INPUT,
+        type: INVALID_METER_VALUE,
+        message: 'Invalid meter value',
+        messageForUser: 'api.meter.registerMetersReadings.INVALID_METER_VALUE',
+        messageInterpolation: { key, value },
+    }),
+}
+
+function toISO (str) {
+    return dayjs(str).toISOString()
+}
+
+/**
+ * @param value
+ * @return {undefined|string|null}
+ */
+function normalizeMeterValue (value) {
+    if (!(isString(value) || isNumber(value) || isUndefined(value))) return null
+    if (isUndefined(value)) return value
+
+    const transformedValue = String(value).trim().replaceAll(',', '.')
+    if (isEmpty(transformedValue)) return undefined
+    return String(Number(transformedValue))
+}
+
+/**
+ * @param {string | null | undefined} value
+ * @return {boolean}
+ */
+function validateMeterValue (value) {
+    if (!isString(value) && !isUndefined(value)) return false
+    if (isUndefined(value)) return true
+
+    return !isEmpty(value) && !isNaN(Number(value)) && isFinite(Number(value)) && Number(value) >= 0
+}
```

Comment:
Does you really need all of these checks? IMO you can leave here the last one 
```suggestion
function validateMeterValue (value) {
    if (!isString(value) && !isUndefined(value)) return false
    if (isUndefined(value)) return true

    return Number(value) >= 0
}
```

Reply:
This logic check data from excel file. I've moved it from the frontend and I'd like to keep it as is.

Reply:
> At a minimum, this case will be valid, although it shouldn’t :) <img alt="Снимок экрана 2024-04-16 в 12 21 53" width="179" src="https://private-user-images.githubusercontent.com/56914444/322741837-4fbe47e9-f5b2-4f3a-b4de-86d06f255aa2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzNTM4NDAsIm5iZiI6MTcxMzM1MzU0MCwicGF0aCI6Ii81NjkxNDQ0NC8zMjI3NDE4MzctNGZiZTQ3ZTktZjViMi00ZjNhLWI0ZGUtODZkMDZmMjU1YWEyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE3VDExMzIyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNmNzU2NGI2ZmUwOTAyNGMwOWY3YjQ0N2RlZjU2MzZjYjgzZTJmNGM2ZjIzMzkyNWIxNWE4Nzg5ODA0ZDRmM2ImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6QPU7oMeThjf6YNsrxliSKG5MGA9ZBLH-GJzWY0STtA">

<img width="247" alt="image" src="https://github.com/open-condo-software/condo/assets/25384290/ce7043d0-b4ec-4c2f-b58c-6bb289bec3c6">